Source/WebCore/html/parser/HTMLEntityParser.cpp

```diff
@@ -146,4 +146,9 @@ size_t decodeNamedEntityToUCharArray(const char* name, UChar result[4])
     return numberOfCodePoints + appendUChar32ToUCharArray(search.mostRecentMatch()->secondValue, result + numberOfCodePoints);
 }
 
+void appendLegalEntityFor(UChar32 c, StringBuilder& decodedEntity)
```

Comment:
Please use the name `character` rather than `c`.

Reply:
OK.

Is it because it is a `UChar32`?

You didn't make the same comment for the other functions taking a `c` so I started wondering.

Reply:
I just ran out of steam reviwing, really. I think occasionally a single letter name could be justified but in a short function I think we’d want to used a word instead of a letter.